### DISCUSSION TITLE
TB source control story

### DISCUSSION THREAD

**Discussion by wqweto** [twinbasic/twinbasic#518] [General]
Compare this:

https://github.com/FullValueRider/AoC2021/tree/main/twinBasic

. . . to this:

https://github.com/FullValueRider/AoC2021/tree/main/Nim

Currently it does not look very good probably because there are no TB sources one can view directly on githuib.

**Comment by WaynePhillipsEA** [comment]
The source is there, it's just buried a little...
https://github.com/FullValueRider/AoC2021/blob/main/twinBasic/ExternalSource/Sources/Day01.twin

**Reply by wqweto** [reply]
Aha, this must be manually exported then.

**Reply by WaynePhillipsEA** [reply]
Yes, he uses Mike's batch script that calls the tb-compiler with the 'export' command:
https://github.com/FullValueRider/AoC2021/blob/main/twinBasic/ExportFromTwinBasic.bat
